client/src/components/molecules/OngoingAgendaCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Box, Card, Text, Button } from "@/components/atoms";
+import { OngoingAgenda } from "biseo-interface/agenda";
+import {
+  ChoiceComponent,
+  CompletedChoice,
+  NotVotableChoice,
+} from "@/components/molecules";
+import { useAgenda } from "@/services/agenda";
+
+type OngoingAgendaProps = {
+  agenda: OngoingAgenda;
+};
+
+export const OngoingAgendaCard: React.FC<OngoingAgendaProps> = ({ agenda }) => {
+  const [chosenChoiceId, setChosenChoiceId] = useState(0);
+  const { voteAgenda } = useAgenda(state => ({
+    voteAgenda: state.voteAgenda,
+  }));
+
+  const vote = () => {
+    if (chosenChoiceId === 0) {
+      return null;
+    } else {
+      voteAgenda(chosenChoiceId, agenda.id);
+    }
+  };
+
+  const choose = (choiceId: number) => {
+    if (choiceId === chosenChoiceId) {
+      setChosenChoiceId(0);
+    } else {
+      setChosenChoiceId(choiceId);
+    }
+  };
+
+  const choices = agenda.user.votable ? (
+    agenda.user.voted ? (
+      <CompletedChoice />
+    ) : (
+      agenda.choices.map((choice, id) => (
+        <ChoiceComponent
+          key={id}
```

Comment:
이 부분도 아래에서 언급한 것과 같이 index 대신 `choice.id`를 쓰는 것이 좋을 것 같아요
```suggestion
      agenda.choices.map(choice => (
        <ChoiceComponent
          key={choice.id}
```